docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,96 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command:
+
+```sh
+npx tinylicious@latest
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by setting an environment variable as `PORT` by running the below command in `Windows Powershell`,
+
+```sh
+$env:PORT=6502
+npx tinylicious@latest
+```
+
+Now, you can navigate to `http://localhost:6502` on your browser to see the Tinylicious service up and running.
+
 ## How to deploy using Tinylicious
 
+The `AzureClient` supports both instantiating against a deployed Azure Fluid Relay service instance for production scenarios, as well as against a local, in-memory service instance, known as Tinylicious, for development purposes.
```

Comment:
```suggestion
The `AzureClient` supports Tinylicious for development purposes in addition to deployed Azure Fluid Relay service instances for production scenarios.
```

@skylerjokiel Should we mention AzureClient + Tinylicious now that we have the local Azure package?